### PR TITLE
Update camera denied permission ui

### PR DESCRIPTION
**Pull request by aminsato**

### CODE REVIEW

File: app/src/main/java/com/vultisig/wallet/ui/screens/ScanQrScreen.kt [hunk 1]
```diff
@@ -78,45 +96,111 @@ internal fun ScanQrScreen(
...
+                    textColor = appColor.oxfordBlue800,
+                    iconColor = appColor.turquoise800,
+                    textStyle = Theme.montserrat.titleLarge,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(
+                            start = dimens.marginMedium,
+                            end = dimens.marginMedium,
+                            bottom = dimens.marginMedium,
+                        ),
```
Comment (YGolybev) [inline app/src/main/java/com/vultisig/wallet/ui/screens/ScanQrScreen.kt:118]: Please, refrain from using dimens; they're breaking the layout. Just specify values from Figma.

File: app/src/main/java/com/vultisig/wallet/ui/screens/ScanQrScreen.kt [hunk 2]
```diff
@@ -78,45 +96,111 @@ internal fun ScanQrScreen(
...
-                    modifier = Modifier.align(Alignment.Center),
+    Scaffold(
+        bottomBar = {
+            if (cameraPermissionState.status.isGranted.not())
+                MultiColorButton(
+                    minHeight = dimens.minHeightButton,
+                    backgroundColor = appColor.turquoise800,
+                    textColor = appColor.oxfordBlue800,
+                    iconColor = appColor.turquoise800,
+                    textStyle = Theme.montserrat.titleLarge,
```
Comment (YGolybev) [inline app/src/main/java/com/vultisig/wallet/ui/screens/ScanQrScreen.kt:111]: Please refrain from using Material aliases for our fonts; that's just misleading

File: app/src/main/java/com/vultisig/wallet/ui/screens/ScanQrScreen.kt [hunk 3]
```diff
@@ -78,45 +96,111 @@ internal fun ScanQrScreen(
...
+                Box(
+                    modifier = Modifier.fillMaxSize(),
+                    contentAlignment = Alignment.Center
+                ) {
+                    Column(horizontalAlignment = Alignment.CenterHorizontally) {
+
+                        Image(
+                            painter = painterResource(id = R.drawable.danger),
+                            contentDescription = null
+                        )
```
Comment (YGolybev) [inline app/src/main/java/com/vultisig/wallet/ui/screens/ScanQrScreen.kt:178]: Please, specify image size

File: app/src/main/java/com/vultisig/wallet/ui/screens/ScanQrScreen.kt [hunk 4]
```diff
@@ -78,45 +96,111 @@ internal fun ScanQrScreen(
...
+
+                        UiSpacer(size = 16.dp)
+                        Text(
+                            text = stringResource(R.string.camera_permission_denied),
+                            textAlign = TextAlign.Center,
+                            color = Theme.colors.neutral100,
+                            style = Theme.montserrat.titleMedium,
```
Comment (YGolybev) [inline app/src/main/java/com/vultisig/wallet/ui/screens/ScanQrScreen.kt:185]: It's menlo.subtitle1 in Figma

**Review by YGolybev — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/vultisig/wallet/ui/screens/ScanQrScreen.kt (hunk 3 above)
Comment (aminsato) [reply app/src/main/java/com/vultisig/wallet/ui/screens/ScanQrScreen.kt]: Done

**Review by YGolybev — APPROVED** [review]

**Pull request merged**